GSoC24/RelationExtraction/methods.py

```
@@ -1,22 +1,49 @@
+        
+        # --- 4. Convert to List of Dictionaries ---
+        triples_data = []
+        for trip in result.triples:
+            triples_data.append({
+                "sub": trip.subject,
+                "rel": trip.predicate,
+                "obj": trip.object
+            })
+            
+        print(f"   Found {len(triples_data)} triples.")
+        return triples_data
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)